README.md

```diff
@@ -117,11 +157,10 @@ These SDKs are fully compatible with the Okta Sign-in Widget and provide utiliti
 
 ### Javascript
 
-- [okta-auth-js](https://github.com/okta/okta-auth-js)
+- [okta-auth-js](https://github.com/okta/okta-auth-js) (Browser or NodeJS)
 - [okta-react](https://github.com/okta/okta-react)
 - [okta-angular](https://github.com/okta/okta-angular)
 - [okta-vue](https://github.com/okta/okta-vue)
-- [oidc-middleware](https://github.com/okta/okta-oidc-js/tree/master/packages/oidc-middleware) (Express / NodeJS)
```

Comment:
Does this deletion mean we are deprecating `oidc-middleware`?

Reply:
De facto, yes. oidc-middleware does not support interaction code flow. The new samples in AuthJS cover the same territory, we have support for Node/Express in AuthJS now.